modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -58,72 +78,72 @@ sshKey: <public_ssh_key> <5>
 +
 [IMPORTANT]
 ====
-The Agent-based Installer uses {op-system-first}. The `rootfs` image, which is mentioned in a subsequent listed item,  is required for booting, recovering, and repairing your operating system. 
+The Agent-based Installer uses {op-system-first}. The `rootfs` image, which is mentioned in a later listed item,  is required for booting, recovering, and repairing your operating system.
 ====
 
-. Configure the `agent-config.yaml` configuration file to meet your organization’s requirements. 
+. Configure the `agent-config.yaml` configuration file to meet your organization’s requirements.
 +
 .Example `agent-config.yaml` configuration file that sets values for an IPv4 formatted network.
 [source,yaml]
 ----
 apiVersion: v1alpha1
 metadata:
-  name: <cluster_name> <1>
+  name: <cluster_name> // <1>
   namespace: <cluster_namespace> <2>
-rendezvousIP:<ip_address_from_CIDR> <3>
-bootArtifactsBaseURL:<server_URL> <4>
-# …
+rendezvousIP: <ip_address_from_CIDR> <3>
+bootArtifactsBaseURL: <server_URL> <4>
+# ...
 ----
-<1> The cluster name that you specified in your DNS record. 
-<2> The name of your cluster on {product-title}. 
-<3> If you are using IPv4 as the network IP address format, ensure that you set the `rendezvousIP` parameter to an IPv4 address that the VCN’s Classless Inter-Domain Routing (CIDR) method allocates on your network. Also ensure that at least one instance from the pool of instances that you booted with the ISO matches the IP address value you set for `rendezvousIP`.  
+<1> The cluster name that you specified in your DNS record.
+<2> The namespace of your cluster on {product-title}.
+<3> If you use IPv4 as the network IP address format, ensure that you set the `rendezvousIP` parameter to an IPv4 address that the VCN’s Classless Inter-Domain Routing (CIDR) method allocates on your network. Also ensure that at least one instance from the pool of instances that you booted with the ISO matches the IP address value you set for `rendezvousIP`.
 <4> The URL of the server where you want to upload the `rootfs` image.
 
 . Apply one of the following two updates to your `agent-config.yaml` configuration file:
 +
-* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the `rootfs` image into the `./<installation_directory>/boot-artifacts` directory on your local system. Upload `rootfs` to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file. 
+* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the `rootfs` image into the `./<installation_directory>/boot-artifacts` directory on your local system. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload `rootfs` to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file.
```

Comment:
AFAIK, rootfs is only marked with backticks when it is used as a directory name. In this context, it is the name of the discovery image, so it should not be surrounded by backticks.

Reply:
Thanks. For a follow-up PR. I need to address merge conflicts on 4.14-4.16